src/mono/mono/mini/aot-compiler.c

```diff
@@ -14355,6 +14392,217 @@ add_interp_in_wrapper_for_sig (MonoAotCompile *acfg, MonoMethodSignature *sig)
 	add_method (acfg, wrapper);
 }
 
+//---------------------------------------------------------------------------------------
+//
+// is_direct_pinvoke_parsable checks whether the direct pinvoke should be parsed into
+// the corresponding module and entrypoint names. Empty lines and comments are skipped.
+//
+// Arguments:
+//  * direct_pinvoke - the string corresponding to the direct pinvoke to parse
+//
+// Return Value:
+//  gboolean pertaining to whether or not the direct pinvoke should be parsed.
+//  into the respective module_name and entrypoint_name.
+//
+
+static gboolean
+is_direct_pinvoke_parsable (char *direct_pinvoke)
```

Comment:
could be const char *

Reply:
Is `const char *` preferred to ensure that the contents of the char data is not changed in this function? Even though nothing in the function would mutate it?

Reply:
Using const char * will make compiler generate compile errors if you try to change the string content in the function and it tells caller of function that function won't change the value.